packages/compass-e2e-tests/helpers/commands/hide-index.ts

```diff
@@ -22,7 +22,7 @@ export async function hideIndex(
     await browser.screenshot(screenshotName);
   }
 
-  await browser.clickVisible(Selectors.ConfirmationModalConfirmButton());
+  await browser.clickVisible(Selectors.confirmationModalConfirmButton());
```

Comment:
drive-by: I cleaned up the naming. It is TitleCase for constants and camelCase for functions.